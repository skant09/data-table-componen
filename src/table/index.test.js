import React from 'react';
import {shallow} from 'enzyme';
import Table from './table';

it('renders without crashing', () => {
  shallow(<Table />);
});