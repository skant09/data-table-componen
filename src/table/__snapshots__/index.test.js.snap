// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  onScroll={[Function]}
  style={
    Object {
      "clear": "both",
      "fontSize": "10px",
      "margin": "0 auto",
      "maxHeight": "400px",
      "overflow": "scroll",
    }
  }
>
  <table
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <thead
      style={
        Object {
          "background": "#ccc",
          "border": "none",
        }
      }
    >
      <tr />
    </thead>
    <tbody />
  </table>
</div>
`;
